pkgs/top-level/python-packages.nix

```diff
@@ -2427,6 +2427,8 @@ self: super: with self; {
 
   dataclasses-serialization = callPackage ../development/python-modules/dataclasses-serialization { };
 
+  dataclass-wizard = callPackage ../development/python-modules/dataclass-wizard { };
```

Comment:
Sorry, this should be before `dataclasses-json`.

Reply:
ah yeah